DatabaseMigrations/2020-04-20-01-UpdatePlayerTable.cs

```diff
@@ -0,0 +1,42 @@
+using FluentMigrator;
+
+namespace simulation.DatabaseMigrations
+{
+    [Migration(2020042001)]
+    public class Migration2020042001 : Migration
+    {
+        public override void Up()
+        {
+            Delete.Table("Player");
```

Comment:
hmmm... I wonder if initially, instead of doing this; we should just manually drop the schema or all the tables to always start with a clean slate until the initial, clean migrations are all ready.

Reply:
As I think about it, I'd like to get rid of this for now. Let's just manually drop the schema/database and recreate it empty before the run as least until we have a decently established schema.